### PR TITLE
Skynet and Substrate Integration

### PR DESCRIPTION
**Pull request by dghelm**
# Grant Application Checklist

- [x] The [application template](https://github.com/w3f/Open-Grants-Program/blob/master/applications/application-template.md) has been copied, renamed ( `project_name.md`) and updated.
- [x] A BTC or Ethereum (DAI) address for the payment of the milestones is provided inside the application.  
- [x] I have read and acknowledged the [Terms and Conditions](https://github.com/w3f/Open-Grants-Program/blob/master/src/T&Cs.md).
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The total funding amount of the project is below USD $30k for initial grant applications and $100k for follow-up grants.
- [x] The initial PR contains only one commit (squash if needed before submitting your PR).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery).

### CODE REVIEW

File: applications/skynet-substrate-integration.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
+# W3F Open Grant Proposal
+
+
+* **Project Name:** Pallet for Decentralized Off-Chain Storage on Skynet
+* **Team Name:** Skynet Labs
+* **Payment Address:** BTC `39aFV1bfcxgjbYrWjgMs3A2fjM7bkKouhg`
```
Comment (mmagician) [inline applications/skynet-substrate-integration.md:6]: This needs to be updated, as we don't award BTC grants anymore.

**Review by mmagician — APPROVED** [review]
Thanks for the extra level of detail, @dghelm. Good luck with the work!

**Review by Noc2 — APPROVED** [review]

**Review by alxs — APPROVED** [review]

**Comment by semuelle** [comment]
Hey @dghelm, could you give another update on the project?

**Comment by dghelm** [comment]
> Hey @dghelm, could you give another update on the project?

Absolutely. When working with off-chain workers pallets, the two devs we have on the project have run into lots of issues running documentation examples, and blocking issues have delayed us a bit. Dan Shields and other folks have gotten us moving again though, and we are now modifying the off-chain worker example pallet with the Substrate code from the repo above and hope to see few issues getting Skynet interaction working, and will publish it alongside documentation when completed.

**Comment by semuelle** [comment]
Thanks for the update, Daniel. Great to hear. I'm familiar with the pain of things moving too quickly for documentation (or oneself for that matter) to keep up. Please keep us in the loop. If the delivery is moving towards February, it'd be best to submit an amendment with an updated timeline. Other than that, keep going. :)

**Comment by dghelm** [comment]
> @dghelm checking in again, any progress? The repo [hasn't really been touched](https://github.com/SkynetLabs/skynet-substrate/commits/main) since October. Are you still interested? As Sebastian mentioned, it's probably time to submit an amendment.

We've made steady progress, but have been slowed down by tooling issues and the project being handed off to another engineer internally. We've got a clear plan towards wrapping up though and should have more code to share soon.

Happy to submit an amendment. Would this just be for the timeline, since we're not looking to alter the scope at the moment? And I can just make a new PR that modifies this file? If you or @semuelle have suggestions, please let me know, and thanks for your patience with the project.

**Comment by alxs** [comment]
That's great to hear. Yes, in that case you can just update the timeline by opening a PR.

**Comment by alxs** [comment]
@dghelm friendly reminder.

**Comment by dghelm** [comment]
> @dghelm friendly reminder.

Thanks. Should be pushing the Grant Milestone Delivery by end-of-day. Will follow up here when submitted.

**Comment by dghelm** [comment]
I've submitted an update explaining time delay and updating company name, emails & payout method in #845 
@alxs

**Pull request merged**